internal/controller/databaseclusterrestore_controller.go

```diff
@@ -451,8 +451,15 @@ func (r *DatabaseClusterRestoreReconciler) restorePG(ctx context.Context, restor
 	if err != nil && !k8serrors.IsNotFound(err) {
 		return err
 	}
+	defer func() {
```

Comment:
So now we have deferred  status update for PG, what about the rest of DBs?

Reply:
The idea of touching the other DBs in the scope of the ticket about the pg restore objects concerns me because it changes the scope, but if you think it's needed - let it be, the change looks pretty harmless. Updated.